src/components/Card/StandardCard/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { useNavigate } from 'react-router-dom';
+
+import { Tile, IconButton } from '@carbon/react';
+import * as Icons from '@carbon/icons-react';
+
+import './styles.scss';
+
+interface StandardCardProps {
+  header: string;
+  description: string;
+  url: string;
+}
+
+const StandardCard = ({
+  header, description, url
+}: StandardCardProps) => {
+  const navigate = useNavigate();
+  return (
+    <Tile className="std-card-main">
+      <div className="std-card-header">
+        <h4 className="std-card-title">{header}</h4>
+        <IconButton className="std-card-button" kind="ghost" label="Go" align="bottom" onClick={() => { navigate(`${url}`); }}>
```

Comment:
This has been a kind of contradictory warning that we need to study a little better before fixing or leaving as it is. Here is an [article](https://reacttraining.com/blog/react-inline-functions-and-performance/) that provides better explanation and can start a discussion. For now, I think we can leave as it is since there are other places on the code like this and come back for a change when we define a course of action.

Also regarding the unit test, it shows an issue when rendering it for the test, it is a tricky issue that will probably be recurrent and need some time do deal with it, so I suggest that we leave as it is now and we open a task to track this.